src/main/java/core/basesyntax/thread/MyThread.java

```diff
@@ -1,5 +1,13 @@
 package core.basesyntax.thread;
 
-public class MyThread {
-    // write your code here
+import java.util.Random;
+
+public class MyThread extends Thread {
+    private static ThreadLocal<Integer> number = new ThreadLocal<>();
+
+    @Override
+    public void run() {
+        number.set(new Random().nextInt(10));
+        System.out.println(number.get());
+    }
```

Comment:
"One of the threads should print out numbers from 0 to 20."  lets implement this task in MyThread class